ocaml/jane/doc/extensions/unboxed-types/index.md

```diff
@@ -284,4 +284,144 @@ Here's the list of primitives that currently support `[@layout_poly]`:
 * `%array_safe_get`
 * `%array_safe_set`
 * `%array_unsafe_get`
-* `%array_unsafe_set`
\ No newline at end of file
+* `%array_unsafe_set`
+
+# Using unboxed types in structures
+
+Unboxed types can usually be put in structures, though there are some restrictions.
+
+These structures support unboxed type fields, but have some restrictions on field
+orders:
+  * Records
+  * Constructors
+  
+Unboxed numbers can't be put in these structures:
+  * Constructors with inline record fields
+  * Exceptions
+  * Extensible variant constructors
+  * Top-level fields of modules
+  * Tuples
+  
+There aren't fundamental issues with the structures that lack support. They will
+just take some work to implement.
+
+Here's an example of a record with some unboxed fields. We call such a record 
```

Comment:
one unboxed field, or add another (or clarify that the int can be in the flat suffix)